pyproject.toml

```diff
@@ -100,3 +100,6 @@ filterwarnings = [
     'ignore::warehouse.utils.exceptions.InsecureOIDCPublisherWarning',
     'ignore::warehouse.packaging.services.InsecureStorageWarning',
 ]
+
+[tool.pip-tools.compile]
```

Comment:
Looks like `--allow-unsafe` will become the default in `pip-tools` 8.x (https://github.com/jazzband/pip-tools/issues/989), so maybe we can add a TODO here?

```suggestion
[tool.pip-tools.compile]
# TODO: This can be removed once it becomes the default.
# See: https://github.com/jazzband/pip-tools/issues/989
```

Reply:
That makes a lot of sense, I had the same thought but forgot to put the comment there. Thanks for the review!